tensorflow/core/kernels/BUILD

```diff
@@ -4365,7 +4365,10 @@ tf_kernel_library(
 tf_kernel_library(
     name = "lrn_op",
     prefix = "lrn_op",
-    deps = NN_DEPS,
+    deps = NN_DEPS
+          + if_rocm([
+           "conv_ops_gpu_hdrs",
```

Comment:
Could you change this to `:conv_ops_gpu_hdrs`?
Also, I think everything fits on one line.

Reply:
done